cli/arguments/port.go

```diff
@@ -57,11 +56,12 @@ func (p *Port) AddToCommand(cmd *cobra.Command) {
 }
 
 // GetPortAddressAndProtocol returns only the port address and the port protocol
-// without any other port metadata obtained from the discoveries. This method allows
-// to bypass the discoveries unless the protocol is not specified: in this
-// case the discoveries are needed to autodetect the protocol.
+// without any other port metadata obtained from the discoveries.
+// This method allows will bypass the discoveries if:
```

Comment:
```suggestion
// This method will bypass the discoveries if:
```

The previous sentence did not make sense. My interpretation was that the removed "allows" word was a leftover from the previous wording.

The alternative would be:

```go
// This method allows bypassing the discoveries if:
```

But I think "will" fits better in this context.